code/modules/mob/transform_procs.dm

```diff
@@ -44,26 +44,37 @@
 
 
 
-//human -> robot
-/mob/living/carbon/human/proc/Robotize()
+/**
+	For transforming humans into robots (cyborgs).
+
+	Arguments:
+	* cell_type: A type path of the cell the new borg should receive.
+	* connect_to_default_AI: TRUE if you want /robot/New() to handle connecting the borg to the AI with the least borgs.
+	* AI: A reference to the AI we want to connect to.
+*/
+/mob/living/carbon/human/proc/Robotize(cell_type = null, connect_to_default_AI = TRUE, mob/living/silicon/ai/AI = null)
 	if(notransform)
 		return
 	for(var/obj/item/W in src)
 		unEquip(W)
-	regenerate_icons()
```

Comment:
This is pointless because the human is going to be immediately transferred into the robot anyway